test/functional/gridfs.test.js

```diff
@@ -3884,4 +3884,29 @@ describe('GridFS', function() {
       });
     }
   });
+
+  /**
+   * @ignore
```

Comment:
I see these scattered throughout the tests inconsistently. Is that the JSDoc `@ignore`, and do we need it in our tests?

Reply:
I have no clue, I removed it for my test.